src/js/component.js

```diff
@@ -220,28 +220,35 @@ class Component {
    * @return {string}
    *         The localized string or if no localization exists the english string.
    */
-  localize(string) {
+  localize(string, tokens, defaultValue = string) {
```

Comment:
yes, we need `defaultValue` as an argument. The `= string` just makes the default value of `defaultValue` to be string. But if a `defaultValue` is passed in to this function, it is used instead of the value of `string.